### ISSUE TITLE
Problem with MDBottomNavigation

### ISSUE THREAD

**Issue by jetsen-os** [HeaTTheatR/Kivy_MVC_Template#1]
Hi.  Beautiful template to implement MVC! It basically works fine, but I get problems when I use the MDBottomNavigation. I think the problem is that MDBottomNavigation uses its own ScreenManager.  Is there a template using MDBottomNavigation?

Thanks

**Comment by HeaTTheatR** [comment]
@jetsen-os I don't see any problems. You can change the template however you like.

**Comment by jetsen-os** [comment]
@HeaTTheatR Yes you are right.  It's more a problem with my understanding of kivy.  Do you have a tip how I can change the template so that I only have one ScreenManager?  Where do I start?  I've tried a few things... Unfortunately without success.  I don't want to change the whole template...
